CMakeLists.txt

```diff
@@ -94,8 +79,6 @@ add_library(ImGui-SFML::ImGui-SFML ALIAS ImGui-SFML)
 target_link_libraries(ImGui-SFML
   PUBLIC
     sfml-graphics
-    sfml-system
-    sfml-window

```

Comment:
Did you remove it because sfml-graphics links to them as well? In my opinion, it's better to link to these modules explicitly as ImGui-SFML uses functions from them

Reply:
It's just wasted code if you ask me but I'll put it back. SFML itself relies on these transitive dependencies so there's strong precedent for doing so.